src/deploy/functions/tasks.ts

```diff
@@ -52,7 +52,7 @@ export function createFunctionTask(
         projectId: params.projectId,
         region: helper.getRegion(fn.name),
         eventType: eventType,
-        functionName: helper.getFunctionName(fn.name),
+        functionName: helper.getFunctionId(fn.name),
```

Comment:
Not for this PR for we should change functionName argument to functionId soon? I volunteer to do that later.

Reply:
Ahhh, this one slipped past me. I like the idea of doing this in a separate PR later - it should be easier to track down any places we missed it then